### PR TITLE
Cards API.

### PR DESCRIPTION
**Pull request by dkorolev**
Hi @mzhurovich ,

This change has plenty of copy-pasting -- unblocking Kolya is more important than technology.

Summary:
- Add card (POST `/ctfo/card/`).
- Get my cards (GET `/ctfo/my_cards`).
- Added the `vote` field to reflect my own vote on the card returned.
- Refactored the unit test.
- Added a shell test to simplify Kolya's life (and because I was lazy to write the real one, it's 2am).

Reference gist: https://gist.github.com/dkorolev/5fb7c7e95e66bfbbd1de

Thanks!
Dima

### CODE REVIEW

File: schema.h [hunk 1]
```diff
@@ -291,6 +291,26 @@ struct Card : yoda::Padawan {
   }
 };
 
+struct CardAuthor : yoda::Padawan {
```
Comment (mzhurovich) [inline schema.h:294]: You could just switch from `Dictionary` to `Matrix` in cards storage...

File: schema.h [hunk 2]
```diff
@@ -398,6 +420,38 @@ struct ResponseFavs {
   }
 };
 
+// "My cards" response schema. Yes, it's the same as favorites. -- D.K.
+struct ResponseMyCards {
+  uint64_t ms;                           // Server timestamp, milliseconds from epoch.
+  ResponseUserEntry user;                // User information.
+  std::vector<ResponseCardEntry> cards;  // Cards created by this used.
```
Comment (mzhurovich) [inline schema.h]: s/used/user/

Comment (dkorolev) [inline schema.h]: :+1: 2b96aa1

File: schema.h (hunk 1 above)
Comment (dkorolev) [inline schema.h:294]: Totally! I thought of it for a couple seconds, looks like the best would be:
1. Have the functionality ready,
2. Have the event replay logic ready, and then
3. Change DB schema, both here and in another place where we're missing the identifier.

Liveable, or am I missing something important? :-)

Comment (mzhurovich) [inline schema.h:294]: Deal!

**Comment by dkorolev** [comment]
:+1: thank you!

**Pull request merged**